src/Slider.jsx

```diff
@@ -667,4 +653,31 @@ class Rheostat extends React.Component {
 Rheostat.propTypes = propTypes;
 Rheostat.defaultProps = defaultProps;
 
-export default Rheostat;
+export default withStyles(({ color, unit }) => ({
+  rheostat: {
+    position: 'relative',
+    overflow: 'visible',
+  },
```

Comment:
nit: it would be nice to leave an empty line between these blocks. https://github.com/airbnb/javascript/tree/master/css-in-javascript#nesting